### PR TITLE
Customize healthcheck query, document embeddings.yaml

### PR DESCRIPTION
**Pull request by pradh**
This is to allow UN SDG to use a different default index with a corresponding health query

### CODE REVIEW

File: deploy/nl/embeddings.yaml [hunk 1]
```diff
@@ -11,6 +11,42 @@
...
+# - usage: what is the model used for?  (EMBEDDINGS, RERANKING)
+# - score_threshold: For embeddings model, what is the cutoff threshold
+#                    below which we drop matches? (default: 0.5)
+#
+# indexes:
+# - store: what type of embeddings store?  (MEMORY, LANCEDB, VERTEXAI)
+# - model: the name of the associated model from `models` section
+# - embeddings: For MEMORY/LANCEDB, the path to the index files.
+#               Can be a local absolute path or GCS (gs://) path.
+# - default_query: if this index were the default index, what is the
```
Comment (shifucun) [inline deploy/nl/embeddings.yaml:30]: maybe healthcheck_query?

File: nl_server/config.py [hunk 2]
```diff
@@ -28,6 +28,9 @@
 NL_EMBEDDINGS_VERSION_KEY: str = 'NL_EMBEDDINGS_VERSION_MAP'
 EMBEDDINGS_SPEC_KEY: str = 'EMBEDDINGS_SPEC'
 
+# Query to use to check index health if this is the default index.
+_DEFAULT_QUERY = 'health'
```
Comment (shifucun) [inline nl_server/config.py:32]: _HEALTHCHECK_QUERY?

**Review by shifucun — APPROVED** [review]
nice!

File: nl_server/config.py (hunk 2 above)
Comment (pradh) [reply nl_server/config.py]: Done.

File: deploy/nl/embeddings.yaml (hunk 1 above)
Comment (pradh) [reply deploy/nl/embeddings.yaml]: Done.

**Comment by pradh** [comment]
Thanks!

**Pull request merged**